wiki/Website-Media.md

```diff
@@ -41,3 +43,44 @@ Regarding the format and settings of the "Export" panel in Figma, please follow
 
 If you have any doubts or problems exporting the files, speak with one of the designers of the HDS team.
+
+## Optimizing images
+
+Prior to adding new images to the website the assets should be optimized to reduce their file size and increase our web performance. Figma's export feature doesn't allow for much fine-grained control regarding image quality and optimizing for speed on the web. [ImageOptim](https://imageoptim.com) works great, is free, easy to use, and with the correct settings can result in a more than _50% reduction_ in file size per image.
+
+1. Install the desktop app for your specific operating system.
+2. Open the application settings; generally most of the default settings work out of the box, but for reference:
+    1. In the quality tab, enable `Lossy minification`
+    2. Set the image qualities for .jpg, .png, and .gif to 85-90%. This results in a high quality, high resolution image (using the guidelines outlined above), while removing unnecessary metadata and compressing the filesize to something more managable for the web.
+3. Within the app, either add images to the queue with the plus icon in the lower left corner, or simply drag and drop files from your file browser into the window.
+
+Once added to the queue assets will be optimized automatically and ImageOptim will automatically save the compressed assets to the same location in the filesystem. **Warning:** This does overwrite the original file.
+
+## Adding and optimizing video assets
+
+Including video examples and prototypes in documentation can help to express complex concepts in a step-by-step manner, but these types of assets require a bit more work to create and subsequently include in markdown. 
+
+_Video assets (.mp4, .mov, etc) aren't directly supported by markdown, for more details on how to include them in documentation view the [markdown documentation](/wiki/Website-Markdown.md)._
+
+These steps are specific to using either Adobe After Effects or Premiere along with Adobe Media Encoder for rendering. These applications are not available by default; you will need to seek approval from your manager and request a license from IT.
+
+These steps are focused on _rendering_ a video asset for use on the web using Adobe Media Encoder. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.
+
+1. Create your source video asset; the most straightforward method is to [record your screen with Quicktime's native screen recording feature](https://support.apple.com/en-gb/guide/quicktime-player/qtp97b08e666/mac) either showing an in-browser example, or Figma prototype.
+2. After launching Media Encoder, add your asset to the render queue.
+    1. In After Effects this can be done through the "Composition" menu and selecting "Add to Media Encoder Render Queue…" (shortcut `Alt` + `Cmd` + `m`).
```

Comment:
```suggestion
    1. In After Effects this can be done through the "Composition" menu by selecting "Add to Media Encoder Render Queue…" (shortcut `Alt` + `Cmd` + `m`).
```